instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -81,14 +81,27 @@ public Span getCurrentSpan() {
   }
 
   protected final boolean shouldStartSpan(Kind proposedKind, Context context) {
+    boolean result;
```

Comment:
Rename `result` to `suppressed` and remove some of negations?

Reply:
done